dnf5daemon-server/callbacks.cpp

```diff
@@ -155,7 +155,7 @@ bool DbusRepoCB::repokey_import(
     bool confirmed;
     try {
         auto signal = create_signal(dnfdaemon::INTERFACE_REPO, dnfdaemon::SIGNAL_REPO_KEY_IMPORT_REQUEST);
-        signal << id << user_id << fingerprint << url << timestamp;
+        signal << id << user_id << fingerprint << url << static_cast<int64_t>(timestamp);
```

Comment:
This is probably because the implementation of the `<<` operator for `sdbus::Message`. Without the explicit conversion it throws an error on `i686` due to ambiguous overload.